### ISSUE TITLE
get_list api does not support additional_fields

### ISSUE THREAD

**Issue by theHeadMike** [klaviyo/klaviyo-api-ruby#10]
We are trying to use the `get_list` API to get a count of the number of profiles in a list, but `profile_count` is not returned.

The [get_list](https://developers.klaviyo.com/en/reference/get_list) API endpoint v2023-06-15 does support query params with `additional_fields`, but the ruby gem does not support these.  

Sample ruby code:
```ruby
opts = {
      additional_fields_list: ["profile_count"]
}
KlaviyoAPI::Lists.get_list("abcde", opts)
```

The call is successful, and we do get the list in the response, but without the additional fields requested.  Output when debugging has been enabled on the gem:
```
ETHON: performed EASY effective_url=https://a.klaviyo.com/api/lists/abcde/ response_code=200 return_code=ok total_time=0.257708
```

Looking in lists_api.rb, neither [get_list](https://github.com/klaviyo/klaviyo-api-ruby/blob/48f3f8cfd7b9c42c2684d744d624aaf85fb6d93a/lib/klaviyo-api-sdk/api/lists_api.rb#L315C11-L315C11) nor [get_list_with_http_info](https://github.com/klaviyo/klaviyo-api-ruby/blob/48f3f8cfd7b9c42c2684d744d624aaf85fb6d93a/lib/klaviyo-api-sdk/api/lists_api.rb#L326) accept the `additional_fields` options.

klaivyo-api-ruby version: 3.0.0
API endpoint version:v2023-06-15

**Comment by dano-klaviyo** [comment]
Hi @theHeadMike , this should be solved in the `4.0.0` version of the gem

```
opts = {
  additional_fields_list: ["profile_count"]
}

response = KlaviyoAPI::Lists.get_list("a-list-id", opts)

pp(response)
```

i get the following response:

```
{
  "data": {
    "type": "list",
    "id": "a-list-id",
    "attributes": {
      "name": "Sample Data List",
      "created": "2023-01-23T18:20:17+00:00",
      "updated": "2023-01-23T18:20:17+00:00",
      "profile_count": 10
    },
    "relationships": {
      "profiles": {
        "links": {
          "self": "https://a.klaviyo.com/api/lists/a-list-id/relationships/profiles/",
          "related": "https://a.klaviyo.com/api/lists/a-list-id/profiles/"
        }
      },
      "tags": {
        "links": {
          "self": "https://a.klaviyo.com/api/lists/a-list-id/relationships/tags/",
          "related": "https://a.klaviyo.com/api/lists/a-list-id/tags/"
        }
      }
    },
    "links": {
      "self": "https://a.klaviyo.com/api/lists/a-list-id/"
    }
  }
}
```

**Comment by theHeadMike** [comment]
Thanks, works as expected.
